cicd/checkMetaMaskIsLatest.ts

```diff
@@ -0,0 +1,37 @@
+import { readFileSync } from 'graceful-fs'
+import * as child_process from 'child_process'
+
+async function main() {
+  const metaMaskReleases = child_process.spawnSync('gh', [
+    'release',
+    'list',
+    '-R',
+    'MetaMask/metamask-extension'
+  ])
+  const metaMaskReleasesStdOut = metaMaskReleases.stdout?.toString()
+  const latestVersion = metaMaskReleasesStdOut.match(
```

Comment:
What do you think of using this package here to check for the versions? 
https://www.npmjs.com/package/semver

It probably will cover more cases than doing it manually. We use it on the client as well and has only one dependency.

Reply:
yeah now that you bring it up, I think we should honestly just check if we're on the same minor version as MetaMask. The patches are updated ~1 per week but the minor versions are ~1 per month.